PerformanceEfficiency/AppServiceAutoscalingSample/Readme.md

```diff
@@ -69,23 +69,77 @@ The duration is set to 5 minutes. This is amount of time that Autoscale engine w
+
+    Loop count setting: you can select a high number of loops, or set it to infinite. If you choose "infinite" you can stop the run with the stop button in the UI.
+
+    Same user on each iteration setting: When you select this checkbox the cookies that you get in the first response will be used for the following requests.
+
+    Delay Thread Creation Until Needed: If this option is checked, the ramp-up delay and startup delay are performed before the thread data is created. If not checked, all the data required for the threads is created before starting the execution of a test
+    
+    Specify thread lifetime: Use this setting if you want to schedule and manage the thread's lifetime
+
+3) Adding the HTTP sampler
+
+    Right click on the Thread Group in the left tree, select "Add", then "Sampler", then "HTTP Request". Enter a name for the HTTP Request and some description in the comments box. 
+
+    Provide the server name of the App Service you are going to load test, for this scenario, enter  "perfstresswebapp.azurewebsites.net" in the "Server Name or IP" box.
+
+    Enter "/Values" in the Path box, that is the Path of you controller's action that simulates a web application having some delays. In this scenario there are no URL parameters but alternateviley you could add them to the parameters list below.
+
+    Keep the other options default (Http Request should be GET).
+
+4) Adding a View Results Window
+
+    Right click on the HTTP Request added to the left tree and select "Add", then "Listener", then "View Results Tree" so once you run the load test it will show you the results in this window.  
 
-*If you already ran the CPU stress scenario, you can skip steps 1-10 and go to step 11 directly.*
+5) Adding a Response Assertion
+
+    You could also add a response assertion. Right click on the HTTP request on the left tree, select "Add", the "Assertions", then "Response Assertions". In this window you could, for example, add a 200 response assertion. Which will fail the entire test plan if any of the HTTP requests receive a response with a reponse code other than 200. Use the "Patterns to Test" section and click on the "Add" button, just enter 200 and add the row to the list.
+
+5) Saving the Test Plan
+
+    Save you test plan in your desired folder. The extension for JMeter is ".jmx". Your test plan is now ready to run.
+
+
+
+#### Deployment instructions
+
+*If you already ran the CPU stress scenario, you can skip steps 1-10 and go to the JMeter running section below*
 
 #### Run these commands by using the Azure CLI from your computer. You need to run az login to log in to Azure. Make sure that you have a subscription associated with your Azure Account If the CLI can open your default browser, it will do so and load an Azure sign-in page. Otherwise, open a browser page at https://aka.ms/devicelogin and enter the authorization code displayed in your terminal.
```

Comment:
```suggestion
> Run these commands by using the Azure CLI from your computer. You need to run az login to log in to Azure. Make sure that you have a subscription associated with your Azure Account If the CLI can open your default browser, it will do so and load an Azure sign-in page. Otherwise, open a browser page at https://aka.ms/devicelogin and enter the authorization code displayed in your terminal.
```